easybuild/easyconfigs/w/WRF/WRF-4.5.1-foss-2023a-dmpar.eb

```diff
@@ -0,0 +1,45 @@
+name = 'WRF'
+version = '4.5.1'
+buildtype = 'dmpar'
+versionsuffix = '-%s' % buildtype
+
+homepage = 'https://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+github_account = 'wrf-model'
+source_urls = [GITHUB_RELEASE]
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'WRF-4.5.1_netCDF-Fortran_separate_path.patch',
+]
+checksums = [
+    {'v4.5.1.tar.gz': '9d557c34c105db4d41e727843ecb19199233c7cf82c5369b34a2ce8efe65e2d1'},
+    {'WRF-4.5.1_netCDF-Fortran_separate_path.patch':
```

Comment:
Note that this is the last time this patch will work, they've changed netcdf detection in 4.5.2

Reply:
4.5.2 needs
```
preconfigopts = "export NETCDF=$EBROOTNETCDFMINFORTRAN && export NETCDF_C=$EBROOTNETCDF &&"
```

Reply:
added a comment about the patch in https://github.com/easybuilders/easybuild-easyconfigs/pull/18741/commits/3f27cb92fe9bf98145628e9a4809e43c8c7ae53a